syntax = "proto3";

option cc_generic_services = true;

import "google/protobuf/empty.proto";
import "service/DataService.proto";
package apb.api.loginBonus;

option java_package = "jp.co.apb.server.api.proto.loginBonus";
option java_multiple_files = true;
option csharp_namespace = "Art.Framework.ApiNetwork.Grpc.Api.LoginBonus";

message ReceiveStampResponse {
  map<string, .apb.api.data.DiffData> diff_user_data = 99;
}

service LoginBonusService {
  rpc ReceiveStamp(.google.protobuf.Empty) returns (.apb.api.loginBonus.ReceiveStampResponse);
}

